packages/custom-functions-metadata/src/generate.ts

```diff
@@ -177,7 +177,10 @@ export async function generateCustomFunctionsMetadata(
     });
 
     if (functions.length > 0) {
-      generateResults.metadataJson = JSON.stringify({ functions: functions }, null, 4);
+      generateResults.metadataJson = JSON.stringify({
+        functions: functions,
+        allowCustomDataForDataTypeAny: true,
```

Comment:
This is certainly simple enough to turn this on by default. However, I would like to know more about the implication of this. If this is enabled, does this affect custom functions which do not return entity data types? 

If custom functions are being used to return entity data types, I believe this is an object type which relates to [CellValue](https://learn.microsoft.com/en-us/javascript/api/excel/excel.cellvalue). I believe the ideal thing here is to have type info for these types and then it can validate that the object is one of the acceptable types. But it might be easiest to start to just say if `object` type is returned for any custom function, that triggers setting allowCustomDataForDataTypeAny to true.

Reply:
@xai-msft Should we discuss this offline? Please schedule a call if that is more helpful to discuss this. Thanks.

Reply:
> @xai-msft Should we discuss this offline? Please schedule a call if that is more helpful to discuss this. Thanks.

@akrantz Sorry, I was OOF last week. I will schedule a meeting soon.